source/pipeline/profiles.rst

```diff
@@ -0,0 +1,22 @@
+.. _profiles:
+
+Profiles
+========
+
+To select one or multiple profiles, please use the ``-profile`` option. For example:
+
+::
+
+    $> nextflow run tractoflow/main.nf --root input_folder --dti_shells "DTI_SHELLS" --fodf_shells "FODF_SHELLS" -profile macos,fully_reproducible -with-singularity singularity_name.img -resume
+
+Profiles available
+------------------
+
+``macos``
+    When this profile is used, TractoFlow will modify a parameter (scratch) for MacOS users.
+
+``use_cuda``
+    When this profile is used, TractoFlow will use eddy_cuda for Eddy process. This feature is available with NVidia GPUs only.
+
+``fully_reproducible``
```

Comment:
default ?

Reply:
I don't think so because longer.

Reply:
Can you explain what is the default behavior ?